templates/gosu/game.rb

```diff
@@ -0,0 +1,22 @@
+require 'gosu'
```

Comment:
yeah il change to app. wasn't sure what to call it since its not really like a web app but app.rb sounds fine.  I think in gosu if you call it main.rb you can actually package it up but iv never gotten that working before